docs-devsite/vertexai-preview.md

```diff
@@ -80,11 +85,16 @@ The Vertex AI in Firebase Web SDK.
 |  [GroundingMetadata](./vertexai-preview.groundingmetadata.md#groundingmetadata_interface) | Metadata returned to client when grounding is enabled. |
 |  [InlineDataPart](./vertexai-preview.inlinedatapart.md#inlinedatapart_interface) | Content part interface if the part represents an image. |
 |  [ModelParams](./vertexai-preview.modelparams.md#modelparams_interface) | Params passed to [getGenerativeModel()](./vertexai-preview.md#getgenerativemodel_e3037c9)<!-- -->. |
+|  [ObjectSchemaInterface](./vertexai-preview.objectschemainterface.md#objectschemainterface_interface) | Interface for ObjectSchema class. |
 |  [PromptFeedback](./vertexai-preview.promptfeedback.md#promptfeedback_interface) | If the prompt was blocked, this will be populated with <code>blockReason</code> and the relevant <code>safetyRatings</code>. |
 |  [RequestOptions](./vertexai-preview.requestoptions.md#requestoptions_interface) | Params passed to [getGenerativeModel()](./vertexai-preview.md#getgenerativemodel_e3037c9)<!-- -->. |
 |  [RetrievedContextAttribution](./vertexai-preview.retrievedcontextattribution.md#retrievedcontextattribution_interface) |  |
 |  [SafetyRating](./vertexai-preview.safetyrating.md#safetyrating_interface) | A safety rating associated with a [GenerateContentCandidate](./vertexai-preview.generatecontentcandidate.md#generatecontentcandidate_interface) |
 |  [SafetySetting](./vertexai-preview.safetysetting.md#safetysetting_interface) | Safety setting that can be sent as part of request parameters. |
+|  [SchemaInterface](./vertexai-preview.schemainterface.md#schemainterface_interface) | Interface for Schema class. |
+|  [SchemaParams](./vertexai-preview.schemaparams.md#schemaparams_interface) | User-facing params passed to specific Schema static methods. |
+|  [SchemaRequest](./vertexai-preview.schemarequest.md#schemarequest_interface) | Final format for Schema params passed to backend requests. |
+|  [SchemaShared](./vertexai-preview.schemashared.md#schemashared_interface) | Basic <code>Schema</code> properties shared across several Schema-related types. |
```

Comment:
Why is Schema tagged as code here, but not anywhere else?

FWIW - I would prolly recommend tagging it almost everywhere as code.  And ObjectSchema and StringSchema, too.

Reply:
Can actually make them links, so did so.

Reply:
I can't seem to find the corresponding new file in the new set of files?